apps/ejabberd/src/mongoose_lib.erl

```diff
@@ -9,3 +11,18 @@ bin_to_int(<<H, T/binary>>, X) when $0 =< H, H =< $9 ->
     bin_to_int(T, (X*10)+(H-$0));
 bin_to_int(Bin, X) ->
     {X, Bin}.
+
+
+%% @doc Database backends for various modules return ok, {atomic, ok}
+%% or {atomic, []} on success, and usually {error, ...} on failure.
+%% All we need is to log an error if such occurred, and proceed normally.
+-spec log_if_backend_error(any()) -> ok.
+log_if_backend_error(V) ->
+    case V of
+        ok -> ok;
+        {atomic, _} -> ok;
+        {error, E} -> ?ERROR_MSG("Error calling backend - got '~p'", [E]);
```

Comment:
Such an error message is not very meaning full. It'd be good to know what `backend` are we talking about, and what operation from this backend.

Also please try to follow #1248 regarding log messages formatting.